libsoftwarecontainer/src/jobs/jobabstract.cpp

```diff
@@ -57,7 +58,8 @@ void JobAbstract::captureStderr()
 
 int JobAbstract::wait()
 {
-    return waitForProcessTermination(m_pid);
+    m_exitStatus = waitForProcessTermination(m_pid);
+    return m_exitStatus;
```

Comment:
`m_exitStatus` is a bool, not an int. Either make it an int and use it as such (I would prefer that since there are more than two possible exit codes) or use it as a bool.